lms/djangoapps/teams/models.py

```diff
@@ -134,3 +200,22 @@ def user_in_team_for_course(cls, user, course_id):
             False if not
         """
         return cls.objects.filter(user=user, team__course_id=course_id).exists()
+
+    @classmethod
+    def update_last_activity(cls, user, course_id):
+        """Set the `last_activity_at` for both this user and their team in the
+        given course. No-op if the user is not a member of a team in
+        this course.
+        """
+        try:
+            membership = cls.objects.get(user=user, team__course_id=course_id)
```

Comment:
I still think you need to track the commentable associated with the change. For the MVP, a student can only be in a single team per course. But we plan to relax that assumption, so we shouldn't make this code dependent on it. If the student is in more than one team in the course, there may be multiple memberships. We need the commentable to see which team the user was "active" in.